### ISSUE TITLE
Aide_design package bug: printing arrays with units

### ISSUE THREAD

**Issue by CynthiaTChan** [AguaClara/aide_design#14]
Error comes up in 0.0.4: `unsuported format string passed to numpy.ndarray._format_` when I try to print arrays that have units. 

This error can be recreated by opening: 
CEE4540_DC_Solutions > DC Flow Control and Measurement Solution .ipynb > running and viewing the cell under Linear Flow Orifice Meter (LFOM) before question 4. 

The last line in this cell is
`print(HeightLfomOrifices.to(u.cm))` which does not run and shows this error in the image attached. 
<img width="1139" alt="screen shot 2017-10-09 at 11 14 50 pm" src="https://user-images.githubusercontent.com/29286359/31400778-ceb9f040-adbe-11e7-8b53-90e1d2857ed3.png">

**Comment by ethan92429** [comment]
This doesn't appear to be a problem with the AIDE design code, but rather with the way pint is designed to handle numpy. It has to do with how pint formats the numbers in a list. The format string I passed in to pint to ensure correct sigfig production caused a problem with printing. Here is the simplest way to recreate the problem with raw pint:
```
import pint
u=pint.UnitRegistry()
u.default_format = '.4g'
hi = [1,2,3,4,5]*u.m
print(hi)
```
It generates the following error:
```
    print(hi)
  File "/Users/ethankeller/.local/lib/python3.6/site-packages/pint/quantity.py", line 140, in __str__
    return format(self)
  File "/Users/ethankeller/.local/lib/python3.6/site-packages/pint/quantity.py", line 186, in __format__
    mstr = format(obj.magnitude, mspec).replace('\n', '')
TypeError: unsupported format string passed to numpy.ndarray.__format__
```
So it appears the [builtin format function can't take the '.4g' format string](https://stackoverflow.com/questions/24170519/python-typeerror-non-empty-format-string-passed-to-object-format) to use with an ndarray. Unfortunately, I have no way of editing the numpy or pint code. The best I can do is to submit an issue with pint. [Here it is.](https://github.com/hgrecco/pint/issues/559) In the meantime, we have to print any Quantity class lists with (with the previously designed `hi`: print(hi.magnitude, hi.units). That will print the expected output. 

**In summary: use print(array.magnitude, array.units) instead of print(array). An official bugfix should be coming through from pint**

Sorry that's the best I have for now!

**Comment by ethan92429** [comment]
@cc2467 please see the answer I have above... it's not ideal, but certainly workable. You'll have to convey to the students that they need to print arrays by doing this: print(array.magnitude, array.units) instead of simply print(array)
